vms/manage.py

```diff
@@ -2,9 +2,22 @@
 import os
 import sys
 
+
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "vms.settings")
-
-    from django.core.management import execute_from_command_line
-
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError:
+        # The above import may fail for some other reason. Ensure that the
+        # issue is really that Django is missing to avoid masking other
+        # exceptions on Python 2.
+        try:
+            import django
+        except ImportError:
```

Comment:
I dont know why you should have this file change though. Please revert all changes in this file.

Reply:
@tonythomas01 manage.py is restored. Please review.